monitor/grafana/medic-benchmark-dashboard.json

```
@@ -2178,11 +2178,11 @@
               },
               {
                 "color": "yellow",
-                "value": 0.1
+                "value": 0.3
               },
               {
                 "color": "red",
-                "value": 0.3
+                "value": 0.5
```

:wrench: Should be 0.6, right?

```suggestion
                "value": 0.6
```